Frameworks/UIKit/UIViewController.mm

```diff
@@ -1139,9 +1144,7 @@ - (void)presentViewController:(UIViewController*)controller animated:(BOOL)anima
         return;
     }
 
-    UIViewController* oldViewController = self;
-    if (priv->_modalViewController != nil) {
-        oldViewController = priv->_modalViewController;
+    if ([self presentedViewController]) {
```

Comment:
although this file does a lot of both, so feel free to disregard.

---
In reply to: [95265157](https://github.com/Microsoft/WinObjC/pull/1601#discussion_r95265157) [](ancestors = 95265157)